CHANGELOG.md

```diff
@@ -20,6 +20,9 @@
 - \#2663 - Adapt filter count, reflect current result set
 - \#2702 - Show proper info on app creation auth error
 
+### Fixed
+- \#2684 - Running instances are confusing
```

Comment:
It's an UX-nightmare, but certainly not a bug.
Please move it under the changed category, thank you. :)